tasks/managed/filter-already-released-advisory-images/filter-already-released-advisory-images.yaml

```diff
@@ -208,8 +207,10 @@ spec:
 
         # Select the correct secret name
         advisorySecretName="create-advisory-prod-secret"
+        printf "production" | tee "$(results.environment.path)"
         if [ "${foundPendingRepositories}" == "true" ]; then
           advisorySecretName="create-advisory-stage-secret"
+          printf "stage" | tee "$(results.environment.path)"
         fi
```

Comment:
I wonder if it would make more sense to use a normal `if-else` here. It seems weird to print "production" and then "stage", although we have `set +x`, so there will be a lot more in the log anyway, but still. WDYT?

Reply:
I'm resolving this since I went with the suggestion and you are on PTO